googlecode_wiki_to_github_markdown/test/file_tests.py

```diff
@@ -0,0 +1,87 @@
+import difflib
+import unittest
+import os
+
+import googlecode_wiki_to_github_markdown
+
+
```

Comment:
2 between functions and classes, 1 between methods.
https://www.python.org/dev/peps/pep-0008/#blank-lines